### PR TITLE
Sync book store

### LINKED ISSUE #842: Sync book-store
- [x] Run `bin/configlet sync -u -e book-store --yes --docs --filepaths --metadata --tests include` (updates the Markdown files and maybe `tests.toml`)
  - docs: instructions unsynced: book-store
- [x] Drop strict types comments from test file and example code (these are useless)
- [x] Add and sync test meta data to tests (`uuid` / `@testdox` in DocBlocks)
- [x] Decide on adding / adjusting / ordering test cases to match current problem specs
  - book-store: missing 3 test cases

Do not redesign the student's interface or add test cases that would invalidate existing community solutions. These are extra tasks, which should be discussed in advance.

### PR DESCRIPTION
**Pull request by tomasnorre**
Resolves #842 

There are some tests failing, with the example configuration. I had tested with my private implementation too, there some fails too. 

So I don't know if some tests aren't meant for the PHP track or what the deal is yet.

I Haven't made as detailed descriptions on the additional tests, they were more confusing than helpful to me.

### CODE REVIEW

**Comment by tomasnorre** [comment]
q

**Comment by tomasnorre** [comment]
Updated to match how I did it in the go-track, in that way it works.

**Comment by mk-mxp** [comment]
@tomasnorre FYI: I am short of time until December. Please be patient...

File: exercises/practice/book-store/.meta/example.php [hunk 1]
```diff
@@ -1,62 +1,42 @@
...
- * For more info review the Concept on strict type checking in the PHP track
- * <link>.
- *
- * To disable strict typing, comment out the directive below.
- */
-
-declare(strict_types=1);
-
-function total($items)
+function total(array $items): int
```
Comment (mk-mxp) [inline exercises/practice/book-store/.meta/example.php:3]: Please adjust the students stub in `BookStore.php` to match this interface. Also add a comment into the stub, something like:

```
// Note: we expect the total in cents (1$ = 100 cents).
```

**Review by mk-mxp — CHANGES_REQUESTED** [review]
Thanks for all the work! Just a tiny change required.

File: exercises/practice/book-store/.meta/example.php (hunk 1 above)
Comment (tomasnorre) [reply exercises/practice/book-store/.meta/example.php:3]: Will do.

**Review by mk-mxp — APPROVED** [review]
Thanks for your patience. And thanks for your contribution!

**Pull request merged**